upload.py

```diff
@@ -374,6 +375,19 @@ async def do_the_thing(base_dir):
                             await client.add_to_client(meta, "PTP")
                     except:
                         console.print(traceback.print_exc())
+
+            if tracker == "TL":
+                tracker_class = tracker_class_map[tracker](config=config)
+                if meta['unattended']:
+                    upload_to_tracker = True
+                else:
+                    upload_to_tracker = cli_ui.ask_yes_no(f"Upload to {tracker_class.tracker}? {debug}", default=meta['unattended'])
+                if upload_to_tracker:
+                    console.print(f"Uploading to {tracker_class.tracker}")
+                    if check_banned_group(tracker_class.tracker, tracker_class.banned_groups, meta):
+                        continue
+                    await tracker_class.upload(meta)
+                    await client.add_to_client(meta, tracker_class.tracker)
```

Comment:
can we add a search_existing() function in TL.py that returns an empty list and then add TL to the api_trackers instead of all this?

Reply:
I did that originally but realized it resulted in strange behavior in the case of a duplicate torrent:
1. "No dupes found" is printed.
2. TL returns a "duplicate torrent" error, which is printed.